packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -183,13 +188,46 @@ export class FungibleTokenTransferController implements ReactiveController {
     }
   }
 
-  async init(env: Environment): Promise<void> {
+  /**
+   * Filter source and destination networks specified by User
+   * @param whitelistedNetworks
+   * @param network
+   */
+  filterWhitelistedNetworks = (
+    whitelistedNetworks: string[] | undefined,
+    network: Domain
+  ): boolean => {
+    // skip filtering if whitelisted networks are empty
+    if (!whitelistedNetworks?.length) return true;
+
+    return whitelistedNetworks.some(
+      (networkName) => networkName.toLowerCase() === network.name.toLowerCase()
+    );
+  };
+
+  async init(
+    env: Environment,
+    whitelistedSourceNetworks?: string[],
+    whitelistedDestinationNetworks?: string[],
+    whitelistedSourceResources?: string[]
```

Comment:
It would be more ergonomic for us to move this into object `options: {whitelistedSourceNmetworks, whitelistedDestinationNetworks...}` as most IDEs would show you name of the properties

Reply:
good idea 👍